experiments/dht/dht_module.py

```diff
@@ -51,6 +84,22 @@ def on_peer_store_error(failure):
 
         self.overlay.store_peer().addCallbacks(on_peer_stored, on_peer_store_error)
 
+    def record_storage_size(self):
+        """
+        Compute the size of the storage in the DHTOverlay
+
+        :return: None
+        """
+        time_stamp = time.time()
+        total_storage_size = 0
+
+        for key in self.overlay.storage.items:
+            for item in self.overlay.storage.get(key):
+                total_storage_size += getsizeof(item)
```

Comment:
Just `sum()` should be just fine here.

Reply:
Was [this](https://github.com/DanGraur/gumby/blob/d5f6116440be4d1420fe74a90b204150ad519dc4/experiments/dht/dht_module.py#L88) what you had in mind for the `sum()`?